src/raft_server.c

```diff
@@ -6353,7 +6371,9 @@ raft_server_instance_run(const char *raft_uuid_str,
 int
 raft_server_get_leader_ts(struct raft_leader_ts *leader_ts)
 {
-    struct raft_instance *ri = raft_net_get_instance();
+    struct raft_instance *ri = raft_server_global_instance.ri;
+    if (!ri)
+        return -EINVAL;
```

Comment:
It's better to avoid overloading the return codes as the error block just below also returns -EINVAL.  I would choose `-ENODEV` here.

For a full list of the errnos and their descriptions see: `$ man 3 errno`